rest/utilities_testing.go

```diff
@@ -244,7 +244,8 @@ func (rt *RestTester) TestPublicHandler() http.Handler {
 }
 
 type changesResults struct {
-	Results []db.ChangeEntry
+	Results  []db.ChangeEntry
+	Last_Seq interface{}
```

Comment:
Not strictly needed as part of this PR, but will likely be needed in future usage of `rt.WaitForChanges()`